test-d/includes.ts

```diff
@@ -0,0 +1,41 @@
+import {expectType} from 'tsd';
+import {Includes} from '..';
+
+const includesEmptyArray: Includes<[], 'abc'> = false;
+expectType<false>(includesEmptyArray);
+
+const includesSingleItemArray: Includes<['colors'], 'colors'> = true;
+expectType<true>(includesSingleItemArray);
+
+const includesComplexMultiTypeArray: Includes<[
+	{
+		prop: 'value';
+		num: 5;
+		anotherArr: [1, '5', false];
+	},
+	true,
+	null,
+	'abcd'
+], 'abc'> = false;
+expectType<false>(includesComplexMultiTypeArray);
+
+const noExtendsProblem: Includes<[boolean], true> = false;
+expectType<false>(noExtendsProblem);
+
+const objectIncludes: Includes<[{}], { a: 1 }> = false;
+expectType<false>(objectIncludes);
+
+const objectIncludesPass: Includes<[{ a: 1 }], { a: 1 }> = true;
+expectType<true>(objectIncludesPass);
+
+// @ts-expect-error
```

Comment:
This should use the `expectError` assertion method instead.

Reply:
> This should use the `expectError` assertion method instead.

When you say instead, do you mean without the `// @ts-expect-error`, because I couldn't figure out how to use the `expectError` assertion method works without it. Is this how I am supposed to do it?:

```ts
// @ts-expect-error
const oneArgIncludes: Includes<['my', 'array', 'has', 'stuff']> = false;
expectError<Includes<['my', 'array', 'has', 'stuff']>>(oneArgIncludes);
```

Removing the `// @ts-expect-error`, gives an error (as expected):

```console
✖  34:22  Generic type Includes requires 2 type argument(s).
```

Reply:
> When you say instead, do you mean without the // @ts-expect-error

Yes

Reply:
You have to put the type inside the method call to `expectError`. `tsd` only ignores TS error inside the `()`.

Reply:
a10c8b5277cb4e95c05ebc41b2eb8564f0e062b1